.github/workflows/dac.yaml

```diff
@@ -0,0 +1,25 @@
+name: Dashboard-as-Code workflow
+
+on:
+  workflow_dispatch:  # Allow manual triggering
+  push:
+    branches:
+      - main
+      - release/*
+      - snapshot/*
+    tags:
+      - v*
+  pull_request:
+  merge_group:
+jobs:
+  dac:
+    uses: perses/cli-actions/.github/workflows/dac.yaml@v0.2.0-beta.0
+    with:
+      url: https://demo.perses.dev
+      directory: ./dashboards
+      server-validation: true
+      cli_version: v0.51.0-beta.1
+    secrets:
+      username: ${{ secrets.DAC_USR }}
+      password: ${{ secrets.DAC_PSW }}
+      cue-token: ${{ secrets.CUE_TOKEN }}
```

Comment:
@nicolastakashi could you please create this repo secret with the value I sent you in DM? ;)